src/AlgebraicGeometry/ToricVarieties/cohomCalg/special_attributes.jl

```diff
@@ -247,11 +247,21 @@ toric line bundle `l` by use of the cohomCalg algorithm
 julia> dP3 = del_pezzo_surface(NormalToricVariety, 3)
 Normal toric variety
 
-julia> cohomology(toric_line_bundle(dP3, [4, 1, 1, 1]), 0)
+julia> sheaf_cohomology(toric_line_bundle(dP3, [4, 1, 1, 1]), 0)
 12
 ```
 """
-function cohomology(l::ToricLineBundle, i::Int; algorithm::String="cohomCalg")
+function sheaf_cohomology(l::ToricLineBundle, i::Int; algorithm::String="cohomCalg")
```

Comment:
I believe this function is documented, so there is likely an instance in the OSCAR-doc folder for toric varieties that need to be adjusted as well.

We also should deprecate `cohomology(l::ToricLineBundle, i::Int; algorithm::String="cohomCalg")` as well as `all_cohomologies`.